client/modules/product-variant/templates/products/productDetail/productDetail.js

```diff
@@ -153,16 +163,20 @@ Template.productDetail.helpers({
       return ReactionProduct.setProduct(instance.productId(),
         instance.variantId());
     }
+
+    return null;
   },
   tags: function () {
     let product = ReactionProduct.selectedProduct();
     if (product) {
       if (product.hashtags) {
         return _.map(product.hashtags, function (id) {
-          return Tags.findOne(id);
+          return Reaction.Collections.Tags.findOne(id);
```

Comment:
Wasn't `Tags` imported already from Collections so it would just be `Tags.findOne`?